yew/src/virtual_dom/vtag.rs

```diff
@@ -1032,7 +1031,7 @@ mod tests {
         #[cfg(feature = "web_sys")]
         document().body().unwrap().append_child(&parent).unwrap();
 
-        let mut elem = html! { <div class=""></div> };
+        let mut elem = html! { <div class?=("")></div> };
```

Comment:
I have put back this test. Obviously it works also for `class?=""` since class is a String. But it also works when using `()`

Reply:
Just so you know, the question mark syntax is supposed to be reworked before 0.18 (see #1550).

*If you actually prefer the question mark syntax over the more implicit one proposed in the referenced issue then please voice your opinion there (I'm looking for excuses not to change it :P)*